server/api/genres.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const router = express.Router()
+const { Genre, Artwork } = require('../db')
+
```

Comment:
i'm a little surprised to see that your "get all genres" route includes all the products with each genre (so if this site were to scale up, a massive amount of data) but then your get-single-genre does *not* include the associated products. sort of the opposite of what you'd normally expect.